### PR TITLE
E2e/feature/import certificates

### PR DESCRIPTION
**Pull request by vlad-dargel**
tests for:
https://redislabs.atlassian.net/browse/RI-3728
https://redislabs.atlassian.net/browse/RI-3684
https://redislabs.atlassian.net/browse/RI-3801

### CODE REVIEW

File: tests/e2e/common-actions/databases-actions.ts [hunk 1]
```diff
@@ -45,5 +54,6 @@ export type ImportDatabaseParameters = {
     userName?: string,
     password?: string,
     connectionType?: string,
-    fileName?: string
+    fileName?: string,
+    parsedJson?: any
```
Comment (ofersteinberg) [inline tests/e2e/common-actions/databases-actions.ts:58]: Please add description

File: tests/e2e/test-data/certs/certsByPath/clientPath.crt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+-----BEGIN CERTIFICATE-----
+mockedClientCrtPath
+-----END CERTIFICATE-----
```
Comment (ofersteinberg) [inline tests/e2e/test-data/certs/certsByPath/clientPath.crt:3]: Couldn't you use one file for this and the two above?
Same for others which are the same

Comment (vlad-dargel) [reply tests/e2e/test-data/certs/certsByPath/clientPath.crt:3]: no, I'm testing here import of certificates by path(on machine), there are cases with the same certificates body and different certificates names and with different body and the same names

that's why we need multiple files here

File: tests/e2e/common-actions/databases-actions.ts (hunk 1 above)
Comment (vlad-dargel) [reply tests/e2e/common-actions/databases-actions.ts:58]: added, thanks!

**Review by nmammadli — APPROVED** [review]

**Pull request merged**